src/chigame/games/models.py

```diff
@@ -34,13 +36,40 @@ class Game(models.Model):
     min_playtime = models.PositiveIntegerField(null=True, blank=True)
     max_playtime = models.PositiveIntegerField(null=True, blank=True)
 
-    complexity = models.PositiveSmallIntegerField(null=True, blank=True)  # 1-5, 1 being the easiest
+    complexity = models.DecimalField(
+        max_digits=3, decimal_places=2, null=True, validators=[MinValueValidator(1), MaxValueValidator(5)]
+    )
+
     category = models.ManyToManyField("Category", related_name="games", blank=True)
     mechanics = models.ManyToManyField("Mechanic", related_name="games", blank=True)
 
     # ================ OTHER ================
     BGG_id = models.PositiveIntegerField(null=True, blank=True)  # BoardGameGeek ID
 
+    # ================ VALIDATON ================
+    def clean(self):
+        # Ensures min_players is not greater than max_players
+        if self.min_players and self.max_players and self.min_players > self.max_players:
+            raise ValidationError({"min_players": "min_players cannot be greater than max_players"})
+
+        # Ensures min_playtime is not greater than max_playtime and that expected_playtime is in this range if provided
+        if self.min_playtime is not None and self.max_playtime is not None:
```

Comment:
In case where there is `min_playtime` but no `max_playtime`, need to check `min_playtime < expected_playtime`. Likewise in case where there is `max_playtime` but no `min_playtime`.

Maybe restructure as follows:
Check if there is `expected_playtime`.
- If there is, then check `min_playtime < expected_playtime` if there is `min_playtime`, and check `expected_playtime < max_playtime` if there is `max_playtime`
- If there isn't, then check `min_playtime < max_playtime` if there is both `min_playtime` and `max_playtime`